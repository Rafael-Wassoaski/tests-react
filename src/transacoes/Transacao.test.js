import {render} from "@testing-library/react";
import React from "react";
import Transacao from "./Transacao";

describe('Componente de transacao do extrato', () => {
    it('Snaphot do componente deve permanecer sempre o mesmo', () => {
        const {container} = render(<Transacao data={'08/09/2021'} tipo={'saque'} valor={'20'}/>);

        expect(container.firstChild).toMatchSnapshot()
    })
})