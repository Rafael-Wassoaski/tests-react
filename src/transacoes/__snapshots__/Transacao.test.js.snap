// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de transacao do extrato Snaphot do componente deve permanecer sempre o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    08/09/2021
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    20
  </p>
</div>
`;
